library/oneview_storage_volume_attachment.py

```diff
@@ -88,7 +88,7 @@ def __init__(self):
         }
 
         super(StorageVolumeAttachmentModule, self).__init__(additional_arg_spec=argument_spec)
-        self.resource_client = self.oneview_client.storage_volume_attachments
+        self.set_resource_object(self.oneview_client.storage_volume_attachments)
 
     def execute_module(self):
 
```

Comment:
attachment = self.oneview_client.storage_volume_attachments.remove_extra_presentations(data)
can be 
attachment = self.recource_client.remove_extra_presentations(data)